perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -123,6 +131,35 @@ sub lookup_site_code_config {
 sub open311_extra_data_include {
     my ($self, $row, $h, $contact) = @_;
 
+    if ($contact->email =~ /Jadu/) {
+        return [
+            {
+                name => 'report_url',
+                value => $h->{url}
+            },
+            {
+                name => 'title',
+                value => $row->title
+            },
+            {
+                name => 'description',
+                value => $row->detail
+            },
+            {   # TODO: Populate from address.
```

Comment:
I guess having these at the FMS end might make sense for them to look at in the admin, though there's also the possibility of doing the lookup at the open311-adapter end if that makes more sense.

Reply:
Yeah I've been going back and forth on this but probably doing it in open311-adapter where we also have the mapping to hand probably makes more sense - dropping these attributes now